projects/plugins/backup/src/js/selectors/assets.js

```diff
@@ -1,5 +1,6 @@
 const assetsSelectors = {
 	getAssetBuildUrl: state => state.assets.buildUrl || null,
+	getConnectedPlugins: state => state.assets.connectedPlugins || [],
```

Comment:
Again, I'm not very familiar with the structure of this code base, but I'm wondering if assets.js is the right place to add this, or if perhaps it should be in it's own file like some of the other selectors in the same directory. Maybe it is a sort of catchall for these types of selectors, I'm not sure, but I wanted to bring it up in case there's better place.

Reply:
Good call, I have moved connectedPlugins to be its own un-nested property on the initial state object and added a separate selector/ reducer file to handle that as part of loading the redux state.